plugins/CoreHome/vue/src/Matomo/Matomo.ts

```diff
@@ -65,8 +65,10 @@ piwik.postEvent = function postMatomoEvent(
   piwik.postEventNoEmit(eventName, ...args);
 
   // required until angularjs is removed
-  const $rootScope = piwik.helper.getAngularDependency('$rootScope') as any; // eslint-disable-line
-  return $rootScope.$oldEmit(eventName, ...args);
+  angular.element(() => {
```

Comment:
@diosmosis seems this broke the js tests. See https://app.travis-ci.com/github/matomo-org/matomo/jobs/547700442#L2281-L2291
does there maybe need to be a `return` before the `angular.element`? Or the return type needs to be adjusted

Reply:
No, we can't return anything here unfortunately. It doesn't seem like the result is used in angular.js so it should be fine. I've included it in another PR, but will extract the fix into a new one.

Reply:
Not returning anything is fine as well. Just wasn't sure how this needs to be fixed